tonic/src/status.rs

```diff
@@ -579,8 +579,8 @@ impl Status {
     }
 
     /// Build an `http::Response` from the given `Status`.
-    pub fn into_http(self) -> http::Response<BoxBody> {
-        let mut response = http::Response::new(crate::body::empty_body());
+    pub fn into_http<B: Default>(self) -> http::Response<B> {
```

Comment:
Maybe it makes sense to add another method for this instead, so that this change could be made in a semver-compatible release? (I guess it depends on your context whether that's useful/feasible.)

Reply:
That is useful and feasible, but I think breakage is relatively unlikely, since `BoxBody` will be type-inferred in most cases. Even if there is breakage, it is going to be in more advanced usage e.g. middleware.

Reply:
This PR is currently marked as a breaking change. If you think that is not justified, I think it would be good to make that argument in a bit more detail.

Reply:
According to [RFC 1105](https://rust-lang.github.io/rfcs/1105-api-evolution.html), this may be treated as a minor change.

Reply:
According to [RFC 1105](https://rust-lang.github.io/rfcs/1105-api-evolution.html), this might be treated as a minor change.